src/mon/PGMap.cc

```diff
@@ -2817,6 +2817,13 @@ void PGMap::get_health_checks(
     checks->add("OSD_SCRUB_ERRORS", HEALTH_ERR, ss.str());
   }
 
+  // LARGE_OMAP_OBJECTS
+  if (pg_sum.stats.sum.num_large_omap_objects) {
+    ostringstream ss;
+    ss << pg_sum.stats.sum.num_large_omap_objects << " large omap objects";
+    checks->add("LARGE_OMAP_OBJECTS", HEALTH_WARN, ss.str());
```

Comment:
we can indicate which pool(s) the objects are in in the detail strings.  see the POOL_APP error for a similar example